LayoutTests/svg/filters/feComponentTransfer-gamma.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="fuzzy" content="maxDifference=0-172; totalPixels=0-36515" />
```

Comment:
This is a large pixel tolerance. It makes it possible that the test may fail and we do not detect the failure.